feature/experimental/system/health/tests/system_generic_health_check/system_generic_health_check_test.go

```diff
@@ -0,0 +1,745 @@
+		t.Fatalf("ERROR: %s is not present", query)
+	}
+
+	for _, drop := range asicDrops {
+
+		if drop.InterfaceBlock != nil {
+
+			if drop.InterfaceBlock.InDrops != nil && drop.InterfaceBlock.GetInDrops() > 0 {
+				t.Errorf("ERROR: InDrops counter got %d, want 0", drop.InterfaceBlock.GetInDrops())
+			}
+
+			if drop.InterfaceBlock.OutDrops != nil && drop.InterfaceBlock.GetOutDrops() > 0 {
+				t.Errorf("ERROR: InDrops counter got %d, want 0", drop.InterfaceBlock.GetOutDrops())
+			}
+
+			if drop.InterfaceBlock.Oversubscription != nil && drop.InterfaceBlock.GetOversubscription() > 0 {
+				t.Errorf("ERROR: InDrops counter got %d, want 0", drop.InterfaceBlock.GetOversubscription())
+			}
+
+		}
+
+		if drop.LookupBlock != nil {
+
+			if drop.LookupBlock.AclDrops != nil && drop.LookupBlock.GetAclDrops() > 0 {
+				t.Errorf("ERROR: AclDrops counter got %d, want 0", drop.LookupBlock.GetAclDrops())
+			}
+
+			if drop.LookupBlock.ForwardingPolicy != nil && drop.LookupBlock.GetForwardingPolicy() > 0 {
+				t.Errorf("ERROR: ForwardingPolicy got %d, want 0", drop.LookupBlock.GetForwardingPolicy())
+			}
+
+			if drop.LookupBlock.FragmentTotalDrops != nil && drop.LookupBlock.GetFragmentTotalDrops() > 0 {
+				t.Errorf("ERROR: FragmentTotalDrops got %d, want 0", drop.LookupBlock.GetFragmentTotalDrops())
+			}
+
+			if drop.LookupBlock.IncorrectSoftwareState != nil && drop.LookupBlock.GetIncorrectSoftwareState() > 0 {
+				t.Errorf("ERROR: IncorrectSoftwareState counter got %d, want 0", drop.LookupBlock.GetIncorrectSoftwareState())
+			}
+
+			if drop.LookupBlock.InvalidPacket != nil && drop.LookupBlock.GetInvalidPacket() > 0 {
+				t.Errorf("ERROR: InvalidPacket counter got %d, want 0", drop.LookupBlock.GetInvalidPacket())
+			}
+
+			if drop.LookupBlock.LookupAggregate != nil && drop.LookupBlock.GetLookupAggregate() > 0 {
+				t.Errorf("ERROR: LookupAggregate counter got %d, want 0", drop.LookupBlock.GetLookupAggregate())
+			}
+
+			if drop.LookupBlock.NoLabel != nil && drop.LookupBlock.GetNoLabel() > 0 {
+				t.Errorf("ERROR: NoLabel counter got %d, want 0", drop.LookupBlock.GetNoLabel())
+			}
+
+			if drop.LookupBlock.NoNexthop != nil && drop.LookupBlock.GetNoNexthop() > 0 {
+				t.Errorf("ERROR: NoNexthop counter is %d, not zero", drop.LookupBlock.GetNoNexthop())
+			}
+
+			if drop.LookupBlock.NoRoute != nil && drop.LookupBlock.GetNoRoute() > 0 {
+				t.Errorf("ERROR: NoRoute counter is %d, not zero", drop.LookupBlock.GetNoRoute())
+			}
+
+			if drop.LookupBlock.Oversubscription != nil && drop.LookupBlock.GetOversubscription() > 0 {
+				t.Errorf("ERROR: Oversubscription counter got %d, want 0", drop.LookupBlock.GetOversubscription())
+			}
+
+			if drop.LookupBlock.RateLimit != nil && drop.LookupBlock.GetRateLimit() > 0 {
+				t.Errorf("ERROR: RateLimit counter got %d, want 0", drop.LookupBlock.GetRateLimit())
+			}
+		}
+	}
+}
+
+func TestInterfaceStatus(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+
+	for _, intf := range interfaces {
+
```

Comment:
Delete blank line

Reply:
Thanks for your comments. Removed extra blank line.
```
	for _, intf := range interfaces {
		query := gnmi.OC().Interface(intf).State()
		root := gnmi.Get(t, dut, query)
```